### PR TITLE
fixed export error for appearances without chunks

### PR DESCRIPTION
**Pull request by manavortex**
# fixed export error for appearances without chunks

### CODE REVIEW

File: WolvenKit.Modkit/RED4/Tools/MeshTools.cs [hunk 1]
```diff
@@ -581,9 +581,10 @@ public static List<RawMeshContainer> ContainRawMesh(MemoryStream gfs, MeshesInfo
...
                 for (var e = 0; e < apps.Count; e++)
                 {
-                    // Null-proof materials
-                    var d = info.appearances[apps[e]];
-                    info.appearances[apps[e]] = InitializeDefaultMaterial(info.meshCount, d.NotNull());
+                    var appearance = apps[e];
+                    // Null-proof materials by making sure we have one material instance per submesh
+                    var d = info.appearances[appearance];
+                    info.appearances[apps[e]] = InitializeDefaultMaterial(info.meshCount, ref d);
                     meshContainer.materialNames[e] = d[index];
```
Comment (seberoth) [inline WolvenKit.Modkit/RED4/Tools/MeshTools.cs:588]: Why does `InitializeDefaultMaterial` now both return the value AND the same data as ref?
Isn't it the same as `meshContainer.materialNames[e] = info.appearances[apps[e]][index];` now?

Comment (manavortex) [reply WolvenKit.Modkit/RED4/Tools/MeshTools.cs:588]: Now that you say it. I'll fix it

Comment (manavortex) [reply WolvenKit.Modkit/RED4/Tools/MeshTools.cs:587]: Updated

**Review by seberoth — APPROVED** [review]

**Comment by manavortex** [comment]
The array doesn't get overwritten anymore, no passing the ref anymore

**Comment by seberoth** [comment]
> The array doesn't get overwritten anymore, no passing the ref anymore

Is this the answer to my comment on the other PR? Also why is it 2 now?
One will conflict with the other

**Comment by manavortex** [comment]
The other one had the wrong title for some reason???

**Pull request merged**